app/pods/components/doubt-view/component.js

```diff
@@ -21,9 +21,14 @@ export default class DoubtViewComponent extends Component {
     return env.discussBaseUrl + '/' + avatar_template.replace('{size}', '150')
   }
 
+  @computed('topicResponse.posts_count')
+  get numberOfPosts () {
```

Comment:
This is misleading. I would rather prefer a computed property which is a boolean as to whether the number of posts is `>1` or not.

An alternate way is to use https://github.com/jmurphyau/ember-truth-helpers (it already used in this project) and use `gt` helper in the template itself.